apps/token-e2e/src/support/common.functions.js

```diff
@@ -69,3 +69,29 @@ Cypress.Commands.add('restartVegacapsuleNetwork', () => {
     })
     .should('contain', 'starting network success');
 });
+
+Cypress.Commands.add('set_up_tokens', (wallet = 0, contract = 0) => {
```

Comment:
set_up_tokens sounds very vague to me. I would suggest renaming it to make it clear its associating vega tokens.

Reply:
Same with `wallet` and `contract`. Maybe `vegaWalletAmount` and `ethWalletAmount` would be better?